src/helm/benchmark/scenarios/scenario.py

```diff
@@ -62,6 +62,9 @@ class Input:
     multimedia_content: Optional[MultimediaObject] = None
     """A single input can consists of multimodal content interleaved (e.g., text, image, text, ...)."""
 
+    original_text: Optional[str] = None
+    """Unperturbed text before any perturbations are applied."""
```

Comment:
I believe that's only possible if `should_include_original_eval` of `DataAugmenterSpec` is `True`, but by default, it's `False`: https://github.com/stanford-crfm/helm/blob/main/src/helm/benchmark/augmentations/data_augmenter.py#L91. 

Even if `should_include_original_eval` is set to `True`, it's inconvenient to find the original instance for a particular perturbed instance and get the original text on the metric level because we usually compute metrics instance by instance (most metrics implement `evaluate_generation`, for example). So, instead of `original_text`, I added a new field `original_instance: Optional[Instance]` to `Instance`. This solution is general enough that it should work for the multimodal case (e.g., perturbed vs. original images). What do you think about this?